interface/resources/qml/controls/WebView.qml

```diff
@@ -37,13 +38,13 @@ Item {
     }
     */
 
-    property alias viewProfile: webroot.webViewCoreProfile
+    // property alias viewProfile: webroot.webViewCoreProfile
```

Comment:
was this needed?

Reply:
I did a searcheroo for that stuff and didn't find anything. I feel like it's a leftover and didn't do anything except seemingly get in the way of my web engine. I wasn't able to get Git Blame working at the time (reset my PC) to take a look at the history of the lines.

I'll take a look again.

Reply:
Looking at the blames, it's entirely possible that was being used to bidirectionally set a profile... Maybe for file selection? No idea. I mean, my useragents and profiles work now as expected. So I'll add to the test plan to ensure nothing stopped working.

Reply:
In order for the this PR to work, is it necessary to comment out this and the other `webViewCoreProfile` lines? (If so they should be deleted, or perhaps they can be uncommented?)

I see that there are other occurrences of `webViewCoreProfile` not addressed in this PR in:
- controls\TabletWebScreen.qml
- controls\TabletWebView.qml

Reply:
I removed them from the stuff I was working on, but after removing them from other stuff, there were some mildly unintended side effects which I think I know how to fix but reallyyyy don't wanna take more time going through those until I have to. heh.

Reply:
These lingering occurrences of `webViewCoreProfile` / side-effects make me nervous. How about filing an issue to capture what you know and as a reminder of this potential issue?

Reply:
Do you mean for the ones that I've touched in this PR or the ones unrelated to the PR? I really should not have touched ones unrelated to this feature so if I did then please mark those? c:

Otherwise, ones I did not touch... they should still be the way they were. The only things I noticed funny were that history would persist restarts (showing it was being saved when it should not be.) Or at least, that's all I remember.

In the future the Interface should offer options if a user wants that kind of functionality but yeah, for now, I didn't really want to mess with it because it could be easy, could be hard. lol